src/tests/test_buildercore_cfngen.py

```diff
@@ -35,3 +35,9 @@ def test_validation_elife_projects(self):
             sleep(0.25)
 
         self.switch_in_test_settings()
+
+    def test_empty_template_delta(self):
```

Comment:
Tried doing this, the problematic step is `update CFN template` as I can update the old one but not the new one (that is always generated). Therefore the test sounds like `remove stuff from CFN template, check that the delta adds them` which is also confusing. Thinking again...